### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by EgorBu**
* add industry / academia badge for conferences
* add ml4p conference
* add code2vec and related article

Signed-off-by: egor <egor@sourced.tech>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,14 +40,15 @@ A curated list of awesome machine learning frameworks and algorithms that work o
...
-* [Workshop on NLP for Software Engineering](https://nl4se.github.io/)
-* [SysML](http://www.sysml.cc/)
+* <img src="badges/origin-industry-green.svg" alt="origin-industry" align="top"> [Machine Learning for Programming](http://ml4p.org/)
+* <img src="badges/origin-academia-green.svg" alt="origin-academia" align="top"> [Workshop on NLP for Software Engineering](https://nl4se.github.io/)
+* <img src="badges/origin-industry-green.svg" alt="origin-industry" align="top"> [SysML](http://www.sysml.cc/)
     * [Talks](https://www.youtube.com/channel/UChutDKIa-AYyAmbT45s991g/)
-* [Mining Software Repositories](http://www.msrconf.org/)
-* [AIFORSE](aiforse.org)
-* [source{d} tech talks](https://talks.sourced.tech/machine-learning-2017/)
+* <img src="badges/origin-industry-green.svg" alt="origin-industry" align="top"> [Mining Software Repositories](http://www.msrconf.org/)
```
Comment (vmarkovtsev) [inline README.md]: MSR is academical for sure

File: README.md [hunk 2]
```diff
@@ -40,14 +40,15 @@ A curated list of awesome machine learning frameworks and algorithms that work o
...
+* <img src="badges/origin-industry-green.svg" alt="origin-industry" align="top"> [AIFORSE](aiforse.org)
+* <img src="badges/origin-industry-green.svg" alt="origin-industry" align="top"> [source{d} tech talks](https://talks.sourced.tech/machine-learning-2017/)
     * [Talks](https://www.youtube.com/playlist?list=PL5Ld68ole7j3iQFUSB3fR9122dHCUWXsy)
-* [NIPS Neural Abstract Machines and Program Induction workshop](ucmlr.github.io/nampi)
+* <img src="badges/origin-academia-green.svg" alt="origin-academia" align="top"> [NIPS Neural Abstract Machines and Program Induction workshop](ucmlr.github.io/nampi)
```
Comment (vmarkovtsev) [inline README.md:51]: Can you please move it before AIFORSE

File: README.md [hunk 3]
```diff
@@ -40,14 +40,15 @@ A curated list of awesome machine learning frameworks and algorithms that work o
 
 ## Conferences
 
-* [Workshop on NLP for Software Engineering](https://nl4se.github.io/)
-* [SysML](http://www.sysml.cc/)
+* <img src="badges/origin-industry-green.svg" alt="origin-industry" align="top"> [Machine Learning for Programming](http://ml4p.org/)
```
Comment (vmarkovtsev) [inline README.md]: I think ml4p is academical

**Review by vmarkovtsev — CHANGES_REQUESTED** [review]

**Comment by vmarkovtsev** [comment]
@EgorBu Do we add the dataset of github issues (#38) here?

**Comment by EgorBu** [comment]
The dataset of GitHub issues (#38) was added in one of the previous PRs

File: README.md (hunk 2 above)
Comment (EgorBu) [reply README.md:51]: It's in chronological order right now. NIPS was the first huge conference in this area

**Review by vmarkovtsev — APPROVED** [review]

**Pull request merged**